app/models/concerns/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] From the previous reviewer's screenshots, this file should be located in the `app/models` directory, not `app/models/concerns`.